File: Modul 6/Soal 1/PRAK601-2410817210014-ZAVIER GHANI.py
baris, kolom = map(int, input("Masukkan jumlah baris dan kolom: ").split())

elemen = list(map(int, input("Masukkan elemen matriks (dipisahkan spasi): ").split()))

matriks = [elemen[i * kolom:(i + 1) * kolom] for i in range(baris)]

print("Output Matriks:")
for row in matriks:
    print(*row)

File: Modul 6/Soal 1/PRAK601-2410817210014-ZAVIER GHANI.c
#include <stdio.h>

int main() {
    int baris, kolom;
    printf("Masukkan jumlah baris dan kolom: ");
    scanf("%d %d", &baris, &kolom);

    int matriks[baris][kolom];

    printf("Masukkan elemen matriks (dipisahkan spasi): ");
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            scanf("%d", &matriks[i][j]);
        }
    }

    printf("Output Matriks:\n");
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }

    return 0;
}
